### PR TITLE
Fargate ecs module

### PR DESCRIPTION
**Pull request by rmrbest**
Fargate ecs module.

- Cluster
- Task definition
- Service
Container json definitions.
We need make a private function to avoid code duplication for services and task definitions.
Also missing  service discovery between fargate containers.

### CODE REVIEW

File: ecs.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+)
+
+locustSlaveService = ecs.Service(
+    cluster=ecs_cluster.arn,
+    resource_name=format_resource_name(name='slave-eu-west-1'),
+    task_definition=locust_slave_task_definiton.arn,
+    network_configuration={
+        "security_groups" : ["sg-0eae5b78fc45c4b24"],        
+        "subnets" : ["subnet-0409a0a361e0101ad"],
```
Comment (LouisAmon) [inline ecs.py]: Why is this hard-coded ?

File: ecs.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+TODO: Extract that Service as method, and call for each service created to avoid code duplication
+"""
+
+locustMasterService = ecs.Service(
+    cluster=ecs_cluster.arn,
+    resource_name=format_resource_name(name='master-eu-west-1'),
+    task_definition=locust_master_task_definiton.arn,
+    network_configuration={
+        "security_groups" : ["sg-0eae5b78fc45c4b24"],        
+        "subnets" : ["subnet-0409a0a361e0101ad"],
```
Comment (LouisAmon) [inline ecs.py]: Hard-coded

File: locust-master.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    {
+      "containerPort": 5558,
+      "hostPort": 5558
+    }
+  ],
+  "environment": [
+    {
+      "name": "ATTACKED_HOST",
+      "value": "https://nuage.studio"
```
Comment (LouisAmon) [inline locust-master.json]: Shouldn't this be defined in the Locust file ?

File: ecs.py (hunk 1 above)
Comment (rmrbest) [reply ecs.py]: Yes, is for information. It should be a config var or values from vpc module for example.

File: ecs.py (hunk 2 above)
Comment (rmrbest) [reply ecs.py]: Yes, is for information. It should be a config var or values from vpc module for example.

File: locust-master.json (hunk 3 above)
Comment (rmrbest) [reply locust-master.json]: No, from locust docker environment is defined as environment variable. I could reasearch in deep from locustifle but i can't find information.

**Pull request merged**